src/Evolution/Systems/Cce/CMakeLists.txt

```diff
@@ -16,10 +16,11 @@ spectre_target_sources(
   LinearSolve.cpp
   NewmanPenrose.cpp
   PrecomputeCceDependencies.cpp
-  ReadBoundaryDataH5.cpp
   ReducedWorldtubeModeRecorder.cpp
   ScriPlusValues.cpp
   SpecBoundaryData.cpp
+  WorldtubeDataManager.cpp
```

Comment:
alphabet

Reply:
At least I got the `W` after the other items this time >.<